components/rmrk/Gallery/CollectionItem.vue

```diff
@@ -303,9 +301,14 @@ export default class CollectionItem extends mixins(
     }
 
     if (this.searchQuery.listed || checkForEmpty) {
-      params.push({
-        price: { greaterThan: '0' },
-      })
+      const prefix = this.$store.getters.currentUrlPrefix
+      if (prefix === 'bsx') {
+        params.push({ price_gt: '0' })
+      } else {
+        params.push({
+          price: { greaterThan: '0' },
+        })
+      }
```

Comment:
you can use `this.urlPrefix` here instead

& I would change instead 
```ts
if (this.urlPrefix === 'rmrk') {
  params.push({
    price: { greaterThan: '0' },
  })
} else {
  params.push({ price_gt: '0' })
}
```
so we make it work for monsama too

Reply:
ahhh yes you are right